### ISSUE TITLE
Build issue on 1.8.1

### ISSUE THREAD

**Issue by marcvsnet** [aleibovici/cryptopump#41]
Getting an build issue on Go 1.6 or 1.7.5 for 1.8.1, 1.8.0 is fine.

```shell
$ go build .
# github.com/aleibovici/cryptopump/mysql
mysql/mysql.go:1080:6: GetGlobal redeclared in this block
        previous declaration at mysql/mysql.go:1051:123
```

**Comment by aleibovici** [comment]
@marcvsnet I'm compiling in prod with go1.16.2 with no issues. Also just updated to go1.17.5 and was able to compile without issues. Could you try updating to go1.17.5 and perhaps rebasing your branch from Main?

**Comment by aleibovici** [comment]
@marcvsnet thanks for letting me know. I believe I fixed the issue in v1.8.1.1
https://github.com/aleibovici/cryptopump/releases/tag/v1.8.1.1.

**Comment by marcvsnet** [comment]
Thanks for that fix!
